### PR TITLE
added trim to getServerAuthenticationToken

### PR DESCRIPTION
**Pull request by Sravan-yarlagadda**
added trim to getServerAuthenticationToken to remove the leading and trailing spaces

### CODE REVIEW

**Comment by Sravan-yarlagadda** [comment]
Hi @henryju Can you please review the changes.

**Comment by Sravan-yarlagadda** [comment]
@henryju  I have updated the change as per you comments. Added trim in the get method and updated the Junit test method to test the current change.

File: src/main/java/hudson/plugins/sonar/SonarInstallation.java [hunk 1]
```diff
@@ -138,6 +138,9 @@ public String getServerUrl() {
    * @since 2.4
    */
   public String getServerAuthenticationToken() {
+    if (serverAuthenticationToken != null) {
+      return serverAuthenticationToken.trim();
```
Comment (henryju) [inline src/main/java/hudson/plugins/sonar/SonarInstallation.java]: Please simplify this method using `return StringUtils.trimToNull(serverAuthenticationToken);`

**Comment by henryju** [comment]
Merged, thanks

**Pull request merged**